Makefile

```diff
@@ -2,13 +2,15 @@
 
 .PHONY: prereqs build test test_fmt validate-generated fmt lint ci devenv
 
+VERSION=`git describe --always --long --dirty`
```

Comment:
it doesn't print the full sha-1 hash. or is it on purpose? i like the full sha-1, though...